### ISSUE TITLE
svg images does not appear on thumbnail mode

### ISSUE THREAD

**Issue by GRFreire** [GRFreire/sxiv#10]
The pr #9 added support for viewing svg images, but when you enter thumbnail mode, it wont show on the grid.

**Comment by bakkeby** [comment]
Yes so far I haven't seen yet anyone who have added support for displaying svg images in thumbnail mode.
